tests/unittest/auto_deploy/singlegpu/transformations/library/test_gemm_fusion.py

```
@@ -595,3 +596,277 @@ def test_fuse_gemms_mixed_children(dtype: str):
+        return torch.randn(self.batch_size, self.seq_len, self.in_features, **kwargs)
+
+    @property
+    def keys_to_pop(self):
+        return ("q_proj.weight", "k_proj.weight", "v_proj.weight")
+
+    @property
+    def num_gemms_after_fusion(self) -> int:
+        return 1
+
+    def forward(self, x):
+        batch_size, seq_len, _ = x.shape  # non-linear user forces mixed_children path
```

Ruff (0.15.10)

[warning] Unpacked variable `batch_size` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)